mqtt5/README.md

```diff
@@ -6,16 +6,34 @@ This document defines how to describe MQTT 5-specific information on AsyncAPI.
 
 ## Version
 
-Current version is `0.1.0`.
+Current version is `0.2.0`.
 
 
 <a name="server"></a>
 
 ## Server Binding Object
 
-This object MUST NOT contain any properties. Its name is reserved for future use.
+This object contains information about the server representation in MQTT5.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="serverBindingObjectSessionExpiryInterval"></a>`sessionExpiryInterval` | [Schema Object][schemaObject] \| integer | Session Expiry Interval in seconds.
+<a name="serverBindingObjectBindingVersion"></a>`bindingVersion` | string | The version of this binding. If omitted, "latest" MUST be assumed.
 
+This object MUST contain only the properties defined above.
 
+##### Example
+
+```yaml
+servers:
+  production:
+    bindings:
+      mqtt5:
+        sessionExpiryInterval: 60
```

Comment:
Would be cool to show an example using Schema Object too.

Reply:
@fmvilas added another example using Schema Object.